### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

Help me use `git add .` again 😄 

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,6 @@ gon_arm64.json
 
 # We don't want to commit env variables
 *.env
+
+# Tool generated files
+*.idea
```
Comment (threebee) [inline .gitignore:33]: [nit - non-block] add a new line.

**Review by threebee — APPROVED** [review]
LGTM

**Review by andreaangiolillo — DISMISSED** [review]

**Comment by gssbzn** [comment]
Non blocking you may want to look at global git ignores
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Here's mine as an example https://github.com/gssbzn/dotfiles/blob/master/.gitignore_global

**Review by threebee — APPROVED** [review]
LGTM!

**Pull request merged**